metadata-models/src/main/pegasus/com/linkedin/chart/ChartInfo.pdl

```diff
@@ -0,0 +1,52 @@
+namespace com.linkedin.chart
+
+import com.linkedin.common.AccessLevel
+import com.linkedin.common.ChangeAuditStamps
+import com.linkedin.common.DatasetUrn
+import com.linkedin.common.Time
+
+/**
+ * Information about a chart
+ */
+record ChartInfo {
+
+  /**
+   * Title of the chart
+   */
+  title: string
+
+  /**
+   * Detailed description about the chart
+   */
+  description: string
+
+  /**
+   * Captures information about who created/last modified/deleted this chart and when
+   */
+  lastModified: ChangeAuditStamps
+
+  /**
+   * Data sources for the chart
+   */
+  inputs: optional array[ChartDataSourceType]
+
+  /**
+   * Query to build a chart from input datasets
+   */
+  query: optional ChartQuery
```

Comment:
do you want to separate this out into separate aspects? One use case I think think of is if we decide to model query as an entity.

Reply:
Made chart query a separate aspect. Do you think we need more separation?